app/src/main/java/com/nextcloud/talk/utils/message/MessageUtils.kt

```diff
@@ -140,6 +143,26 @@ class MessageUtils(val context: Context) {
                             context.startActivity(browserIntent)
                         }
                     }
+                    else -> {
+                        val spannable = SpannableStringBuilder(messageStringInternal)
+                        val placeholder = "{$key}"
+                        val replacementText = individualHashMap["name"]
+                        var start = spannable.indexOf(placeholder)
+                        while (start != -1) {
+                            val end = start + placeholder.length
+                            spannable.replace(start, end, replacementText)
+                            spannable.setSpan(
+                                ForegroundColorSpan(Color.BLUE),
```

Comment:
Instead of blue, it should be the normal font color, it should be bold as any parameters and in case it would be a link in the future it should be underlined